bot/__init__.py

```diff
@@ -15,7 +15,7 @@
 logger.addHandler(handler)
 logger.setLevel(logging.INFO)
 
-__version__ = "0.5.1"
+__version__ = "0.5.2"
```

Comment:
New feature should usually bump the minor version (the middle one) - the last digit is mainly for bugfixes, but that's not as important I guess.